en/06_people/05_journey_of_your_career.md

```diff
@@ -0,0 +1,68 @@
+## Journey of your Career
+
+### Your Great!
+
+You are pretty sure you are good at what you do, some day's you might even say you are amazing at your job! How do you make sure others know how good you are - I don't mean in the periphery of their mind "they do good work" sort of way, but have your name at the tip of their tongue when new opportunities come up? Is not easy, especially when you are surrounded by others doing great work at the same time; here are some ideas to help you out.
+
+I like lists of three's, short enough to remember and long enough to get some good direction.
+
+#### Short Version
+
+1. Define yourself
+2. Pick an audience
+3. Communicate regularly
+
+#### Define Yourself
+
+This is probably the hardest step, for a simple reason, everyone hates tradeoffs. Very important point, all strategy requires explicit (very clear, no ambiguity) or implicit (not defined, but understood by all) tradeoffs. If there are no tradeoffs, there is a very high likelihood of failing at everything. No one can be good at, or do, everything!
+
+Example: "I work in the public cloud." VS "I am a Google Cloud Platform expert, specializing in high level project architecture and staying informed of the ever changing offerings Google creates, discards, or re-brands."
+
+Try creating a few of these statements, feel them out, see how they fit you.
+
+Some things to consider when making these:
+
+* Ensure you enjoy working in your target area, you may be spending a lot of time there
+* Check to see if there is work where you are aiming!
+    * Wanting to be the best ALGOL developer may not make sense
+* Make sure it aligns with your career ladder somehow
+* Your organization Leveling Guide
```

Comment:
The other bullets are a complete sentence (a direction or command) but this one isn't.